Dockerfile

```diff
@@ -5,24 +5,30 @@ FROM node:16-bullseye-slim as base
 ENV NODE_ENV production
 
 # Install openssl for Prisma
-RUN apt-get update && apt-get install -y openssl
+RUN apt-get update && apt-get install -y openssl curl
 
 # Install all node_modules, including dev dependencies
 FROM base as deps
 
 WORKDIR /myapp
 
-ADD package.json package-lock.json ./
-RUN npm install --production=false
+ADD package.json ./
+ADD package-lock.jso[n] yarn.loc[k] pnpm-lock.yam[l] pnpm-lock.ym[l] .pnpmfile.cj[s] ./
+# the following script will install the dependencies with which ever lock file is available
+ADD scripts/install-deps.sh ./install-deps.sh
+RUN ./install-deps.sh
```

Comment:
Since we have `packageManager` available in `remix.init`, shouldn't we just update `remix.init` to install dependencies & replace the `npm` references with the correct calls?

Reply:
maybe? i'm not completely sold on that

Reply:
This way we keep all the `packageManager` related things in `remix.init` & the user of the stack doesn't need to know about it.

Reply:
I agree. Let's have `remix.init` handle this at project generation time.